daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -400,6 +400,32 @@
             </xs:documentation>
           </xs:annotation>
         </xs:element>
+        <xs:element name="unparseSuspensionWaitOld" type="xs:int" default="100" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>
```

Comment:
I like to put <![CDATA[... ]]> around blocks of doc text like this. Seems to tell tooling not to mess up the line indentation and such.

Reply:
We don't have cdata tags documentation for the other tunables. Should I add it for all of them as well? Indentation also doesn't really matter for these, since it's not test related. Do tools even modify xs:documentation tags? I would expect tools wouldn't even modify those since it feels important to not modify documentation tags--tools can't know if format is important for them.

Reply:
I think tools treat this as just plain old XML. They don't know that it is a specific XML-schema tag. So whitespace is fluid. 

I don't think adding more documentation blocks is necessary for this. I do like using the documentation annotations for block  comments. I think of it like javadoc, sort of.

Reply:
Yeah, I gues many XML tools aren't schema aware.

I'm not suggesting more documentation blocks, I'm wondering if we want to add CDATA to all the other documentation blocks in this file for all the other tunables. We don't currently have them.